[Utils] Add ExecStop= to example systemd service

AaronM-Cloudtek opened the pull request:
With no stop command defined `systemctl stop pivxd` will send SIGTERM
which can cause database corruption. With this change `pivx-cli stop`
will be used to stop the daemon process gracefully.

Mrs-X commented:
So far I've never had database base corruption when I shut down a system while pivxd was running, but stopping it the right way is always better 👍 
Did you test this? To me it looks correct, but tests never hurt.

BTW, is there a specific reason for the `-` (minus) for `ExecStop` ?

AaronM-Cloudtek commented:
Yes I did test on CentOS 7. I was getting that GetBestBlock error pretty consistently anytime the daemon shutdown from SIGTERM. Since I added the ExecStop it comes back up cleanly every time.

Regarding the dash, if you add the ExecStop command without it and stop the service while pivxd is still initializing it exits with code 28 and you get a red error-looking message in the service status, like:

```
Process: 24057 ExecStop=/usr/bin/pivx-cli -conf=/etc/pivx/pivx.conf stop (code=exited, status=28)
```

Although it *does* still stop gracefully I found this to be rather confusing so I added the dash so that the exit code from the stop command is ignored. I was also seeing exit code 203 in some cases which for the life of me I can't even find an explanation for. HTTP 203 maybe...? No idea haha.

Mrs-X commented:
utACK https://github.com/PIVX-Project/PIVX/pull/166/commits

Mrs-X commented:
> I was also seeing exit code 203 in some cases which for the life of me I can't even find an explanation for

If I remember correctly the exit code isn't explicitly set on exit. +1 for my ToDo list.
